### PR TITLE
Feature annotation view clay

### PR DESCRIPTION
**Pull request by claywatkins**
This feature changes how the annotation works on the map. When tapping on the annotation it allows for the user to choose if they want to see more details about the city, add to favorites, or dismiss the popup.

### CODE REVIEW

File: labs-ios-starter/Storyboards/Map/MapViewController.swift [hunk 1]
```diff
@@ -72,11 +72,40 @@ extension MapViewController: CLLocationManagerDelegate {
...
+        return annotationView
+    }
+    
+    func mapView(_ mapView: MKMapView, annotationView view: MKAnnotationView, calloutAccessoryControlTapped control: UIControl) {
+        let ac = UIAlertController(title: locationName!, message: "What would you like to do?", preferredStyle: .alert)
+        ac.addAction(UIAlertAction(title: "Add To Favorites", style: .default, handler: { (_) in
+            //Handle adding to favorites
+        }))
+        ac.addAction(UIAlertAction(title: "Details", style: .default, handler: { (_) in
+            //Handle viewing details
```
Comment (iOSjess) [inline labs-ios-starter/Storyboards/Map/MapViewController.swift:95]: I like the comments as reminders for what needs to be done once the backend is connected. Should these be TODO's?

**Review by iOSjess — APPROVED** [review]
Love the alert view and pop-up pin!

File: labs-ios-starter/Storyboards/Map/MapViewController.swift (hunk 1 above)
Comment (claywatkins) [reply labs-ios-starter/Storyboards/Map/MapViewController.swift:95]: Yes the could be marked as TODO, do you think it's a necessary change? :)

Comment (iOSjess) [reply labs-ios-starter/Storyboards/Map/MapViewController.swift:95]: I think it would be beneficial, yes. :-)

**Pull request merged**